src/pages/dev/amplifier/chain-integration/introduction.mdx

```diff
@@ -1,63 +1,101 @@
-- **Message Relayer**
-    - A service built with any tech stack that listens for events on your chain and relays them to the Axelar network.
-    - Get started with our [Message Relayer Example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier)
-
-
-The [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) will guide you through this process within the current Amplifier DevNet. Contract deployment on Testnet or Mainnet will require governance proposal.
-
-## Integration Process
-Most integrators will follow the following process:
-
-1. Learn & Understand Amplifier
-    - Follow the [chain integration tutorial](/dev/amplifier/chain-integration/integrate-a-chain) to deploy standardize contracts to the DevNet to understand the tools and basics of the process.
-1. Customize and Deploy to DevNet
-    - Build (or instantiate or customize) each of the [required contracts and the relayer](#integration-architecture).
-    - You may choose to change the logic of your verifier contract (for ZK use cases for example) or prover contract.
-1. Whitelist contracts 
-    - For the DevNet [use the form](https://docs.google.com/forms/d/e/1FAIpQLSchD7P1WfdSCQfaZAoqX7DyqJOqYKxXle47yrueTbOgkKQDiQ/viewform) to ask to be whitelisted
-1. Become a [verifier](/validator/amplifier/verifier-onboarding) for your chain.
-    - To test your chain, you will need active verifiers to vote on messages from your chain
-1. Test your integration
-    - Make sure you can send and receive messages on your chain.
-1. Begin audit process
-    - You should audit your on-chain contracts.
-1. Propose to Testnet  
-    - [Make on-chain proposals](/learn/cosmwasm-governance) on the testnet to create your CosmWasm smart contracts.
-        1. Proposal to store your CosmWasm code.
-        1. Proposal to instantiate your contracts.
-    - Make an on-chain proposal to add your chain to the Amplifier router.
-1. Test in Testnet
-    - Ensure your contracts are working as expected.
-    - Verify with developers that they can send messages to and from your chain.
-1. Productionize your relayer
-    - Ensure your relayer is robust, fault-tolerant, and can handle the load of your chain.
-1. Propose to Mainnet
-    - [Make on-chain proposals](/learn/cosmwasm-governance) on Mainnet to create your CosmWasm smart contracts.
-        1. Proposal to store your CosmWasm code.
-        1. Proposal to instantiate your contracts.
-    - Make an on-chain proposal to add your chain to the Amplifier router.
\ No newline at end of file
+## Integration architecture
+
+To connect your chain to the Axelar network via the Interchain Amplifier, you'll need to build or instantiate a [message relayer](/dev/amplifier/chain-integration/introduction#message-relayer) service and the following four smart contracts:
```

Comment:
order of this list doesn't make sense when the order below is different. Maybe delete this paragraph entirely or shorten it?

Reply:
I think the order is correct?

Reply:
gateway, contracts, then relayer

Reply:
Relayer sould come last in this subsection.